clusterloader2/docs/oom_detection.md

```diff
@@ -0,0 +1,48 @@
+# Clusterloader2 - tracking OOMs
+
+In order to track pods OOMs (out of memory) during clusterloader2 tests
+execution, TestMetrics has been augmented with `ClusterOomsTracker` measurement.
+This outputs an appropriate summary containing basic information regarding OOMs
+inside the cluster like the PID of OOMing process or the name of a node where
+the OOM happened.
+
+## Enabling OOMs tracking
+
+Firstly, ensure that the `TestMetrics` measurement is added to `Starting
+measurements` and `Collecting measurements` steps of your test config. Next,
+add `clusterOomsTrackerEnabled` parameter and set it to `true` in both steps
+configuration.
+
+Sample configuration in `Starting measurements` step:
+
+```yaml
+- name: Starting measurements
+  measurements:
+    ...
+    - Identifier: TestMetrics
+      Method: TestMetrics
+      Params:
+        action: start
+        clusterOomsTrackerEnabled: true
+```
+
+Sample configuration in `Collecting measurements` step:
+
+```yaml
+- name: Collecting measurements
+  measurements:
+    ...
+    - Identifier: TestMetrics
+      Method: TestMetrics
+      Params:
+        action: gather
+        clusterOomsTrackerEnabled: true
+```
+
+## Further debugging steps
+
+`ClusterOomsTracker` watches for events emitted by `node-problem-detector` when
+an OOM occurs. Such events contain only a fraction of information that may be
+useful for debugging - for more, check `systemd.log` files of an appropriate
+node for the name of OOMing pod/container or the pod's memory limit, for
+instance.
```

Comment:
I would remove ", for instance."
Nit: memory limit is set on a container, not a pod, but in a pod definition.

Reply:
Done.